src/constants/urls.ts

```diff
@@ -2,7 +2,7 @@ import { BASE_URL, ENVIRONMENT, IS_TEST } from "./env";
 
 export const APIs = {
   aws: "https://kpnxz5rzo2.execute-api.us-east-1.amazonaws.com",
```

Comment:
Just created and setup this one this morning. 
```suggestion
  aws: "https://ap-api@better.giving",
```